### PR TITLE
Use h1 for /auctions page title

### PR DESCRIPTION
**Pull request by eessex**
Minor SEO fix noticed while assembling integration tests, use h1 rather than h2 for page title on /auctions route.

Screen Shot 2019-10-17 at 4 32 10 PM

### CODE REVIEW

File: src/desktop/apps/auctions/templates/index.jade [hunk 1]
```diff
@@ -15,7 +15,7 @@ block body
       .aggregate-page-items-featured
         if currentAuctions.length
           .aggregate-page-items-featured-set
-            h2.auctions-header Current Auctions
+            h1.auctions-header Current Auctions
```
Comment (dleve123) [inline src/desktop/apps/auctions/templates/index.jade]: @eessex Is it bad from an SEO perspective to have multiple h1s? I'm noticing that there are [many on this page](https://github.com/artsy/force/blob/7bf1935125e3ae674d2c49912369e2b8df47f739/src/desktop/apps/auctions/templates/index.jade#L48-L49).

Down to merge assuming that we aren't making an existing SEO problem any worse @eessex

**Comment by dleve123** [comment]
@eessex :shipit: pending that there's no negative impact of having multiple h1s. I don't have much SEO context, but just wanted to confirm before integrating. Re-assigning to you so that you can merge once you consider the question!

File: src/desktop/apps/auctions/templates/index.jade (hunk 1 above)
Comment (eessex) [reply src/desktop/apps/auctions/templates/index.jade]: Excellent point- will update those. Apparently we are no longer dinged for multiple h1’s, but we should be semantic either way and only use them for the page title.

**Comment by eessex** [comment]
This has been updated to use `h1` only for the page title.

**Comment by dleve123** [comment]
#mergeongreen

**Pull request merged**